flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/CreditBasedPartitionRequestClientHandlerTest.java

```diff
@@ -430,23 +431,35 @@ private static void releaseResource(SingleInputGate inputGate, NetworkBufferPool
 	/**
 	 * Returns a deserialized buffer message as it would be received during runtime.
 	 */
-	private static BufferResponse createBufferResponse(
+	private BufferResponse createBufferResponse(
 			Buffer buffer,
 			int sequenceNumber,
-			InputChannelID receivingChannelId,
-			int backlog) throws IOException {
+			RemoteInputChannel receivingChannel,
+			int backlog,
+			CreditBasedPartitionRequestClientHandler clientHandler) throws IOException {
+
 		// Mock buffer to serialize
-		BufferResponse resp = new BufferResponse(buffer, sequenceNumber, receivingChannelId, backlog);
+		BufferResponse resp = new BufferResponse(
```

Comment:
not necessary change if still use previous `receivingChannelId`

Reply:
Reverted the change to only passing the `receivingChannelId`